nsfc-plugin/src/main/java/org/osc/controller/nsfc/api/NeutronSfcSdnControllerApi.java

```diff
@@ -54,27 +44,11 @@
                  SUPPORT_NEUTRON_SFC + ":Boolean=true"})
 public class NeutronSfcSdnControllerApi implements SdnControllerApi {
```

Comment:
do we still support if SUPPORT_NEUTRON_SFC is made false? if not please explicitly add the comment.

Reply:
This attribute becomes a bundle property, which is _telling_ the system that this bundle supports SFC. See
https://github.com/opensecuritycontroller/osc-core/blob/9a960d9ca0c34bc0ee576082194cae65e7d2c18b/osc-server/src/main/java/org/osc/core/broker/model/plugin/ApiFactoryServiceImpl.java#L642